managers/restore.js

```diff
@@ -18,7 +18,11 @@ module.exports.lastUsersRestores = () => restoreAdapter.lastUsersRestores();
 //
 module.exports.historyRestoreByUser = (req) => {
   const username = req.query.username;
-  return restoreAdapter.historyRestoreByUser(username);
+  let limit;
+  if (req.query.limit) {
+    limit = parseInt(req.query.limit, 10);
```

Comment:
Since `limit` is only assigned inside the if statement, I think it needs to be initialized with a default value.